packages/flutter/test/material/slider_test.dart

```diff
@@ -3554,8 +3554,11 @@ void main() {
     final Offset sliderCenter = tester.getCenter(find.byType(Slider));
     final Offset tapLocation = Offset(sliderCenter.dx + 50, sliderCenter.dy);
 
-    // Tap the slider to bring up the value indicator.
-    await tester.tapAt(tapLocation);
+    // Create a mouse gesture object to tap the slider and bring up the value indicator.
+    final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse);
```

Comment:
Actually, instead of doing this here, can you add a `kind` parameter to `tapAt` and `tap` in the `flutter_test` package and pass it into the `createGesture` there?  Then it'll be a more generic solution that can be used by others.

Reply:
@gspencergoog I'm sorry, but it has been one week already. Should I do anything else for the PR?